maintenance.go

```diff
@@ -125,15 +126,18 @@ func (m *MaintenanceCreateParams) FillHostIDs(session *Session) error {
 		return err
 	}
 
+	err = errors.New("Failed to tind ID by host name")
```

Comment:
Typo: "tind".

Please also make this error a global var, maybe `ErrMaintenanceHostNotFound` or something to that effect.

Reply:
done